gobblin-data-management/src/main/java/gobblin/data/management/conversion/hive/publisher/HiveConvertPublisher.java

```diff
@@ -227,6 +269,71 @@ public void publishData(Collection<? extends WorkUnitState> states) throws IOExc
     }
   }
 
+  /**
+   * Method to copy partition parameters from source partition to destination partition
+   * @param completeSourcePartitionName dbName@tableName@partitionName
+   * @param completeDestPartitionName dbName@tableName@partitionName
+   */
+  private void copyPartitionParams(String completeSourcePartitionName, String completeDestPartitionName,
```

Comment:
Can we add couple of tests for these. Might be good to have them to lock in behavior along with increasing coverage / verifying functionality.